src/NServiceBus.AcceptanceTests/Sagas/When_receiving_that_completes_the_saga.cs

```diff
@@ -58,6 +58,7 @@ public void Should_ignore_messages_afterwards()
                         SomeId = c.Id
                     }));
                 })
+                .AllowExceptions()
```

Comment:
Good catch. I blindly added it because the test was throwing but I *think* the problem was an unnecessary `IHandleSagaNotFound`